ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -1,14 +1,187 @@
 import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  activeSigils: { x: number; y: number; typeId: string; npcId: string }[];
+  activeFrontSigils: { x: number; y: number; typeId: string; npcId: string }[];
+  activeExplosions: { x: number; y: number }[];
+  activePythons: PluginCombatantState[];
+  // For Quetz, mechanics which only use two of them always use the two with the highest ID
```

Comment:
Yes and no.

It would track all of them correctly if OverlayPlugin gave the same output as the raidemulator suggests but it doesn't.

Sigil/FrontSigils work but Quetzals and Pythons don't, I'm looking through what is actually being supplied by getCombatants but my tooling is very bad in that regard.